R/evaluate.R

```diff
@@ -167,6 +167,15 @@ evaluate <- function(input,
   watcher$get()
 }
 
+signal_error <- function(cnd) {
+  signalCondition(cnd)
+  if (has_rlang) {
+    rlang::entrace(cnd)
```

Comment:
I wonder if this should directly by `rlang::abort()` instead of `rlang::entrace()` so that it really stops with traceback printed. 

This is similar to what have been discussed in https://github.com/r-lib/evaluate/pull/226

Reply:
You can't pass an existing condition object to `abort()` like you can with `stop()`. My read of the rlang docs does suggest that this is what `entrace()` does, so I think we'll need to get @lionel- to help.

Reply:
I've updated with something I tried locally that should at least avoid the `!continue` problem.